BraintreeCore/src/main/java/com/braintreepayments/api/AnalyticsEventParams.kt

```diff
@@ -0,0 +1,12 @@
+package com.braintreepayments.api
+
+import androidx.annotation.RestrictTo
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+data class AnalyticsEventParams constructor(
+    var payPalContextId: String?,
+    var linkType: String?,
+    var isVaultRequest: Boolean,
+) {
+  constructor() : this(null, null, false)
```

Comment:
```suggestion
  // TODO: this is a convenience constructor for Java; remove after Kotlin migration is complete
  constructor() : this(null, null, false)
```
Adding a note to remove this in the future.